packages/dialog/src/base-types.ts

```
@@ -0,0 +1,37 @@
+
+type BaseDialogTriggerProps = {};
+
+type BaseDialogPortalProps = Pick<DialogPortalProps, 'forceMount' | 'children'>;
+
+type BaseDialogOverlayProps = Pick<DialogOverlayProps, 'forceMount'>;
+
+type BaseDialogCloseProps = {};
+
+type BaseDialogTitleProps = {};
+
+type BaseDialogDescriptionProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)